extensions/wikia/PageShare/PageShareHooks.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class PageShareHooks {
+
+	/**
+	 * Add global JS variable wgAcceptLangList
+	 *
+	 * @param array $vars global variables list
+	 * @return boolean return true
+	 */
+	public static function onMakeGlobalVariablesScript(Array &$vars) {
+		global $wgRequest;
+
+		$wgAcceptLang = $wgRequest->getAcceptLang();
+		$vars['wgAcceptLangList'] = is_array( $wgAcceptLang ) ? array_keys( $wgAcceptLang ) : null;
```

Comment:
And based on skin AFAIK, but If we're not vary-ing cache based on Accept-Language, then you're right.